FitNesseRoot/PlugIns/FitServers/DelphiFit/content.txt

```diff
@@ -6,9 +6,6 @@ Updated to FIT20070619 and extended by Michal Wojcik: mgwojcik@gmail.com
 !3 Downloads:
 Project moved to http://code.google.com/p/fit4delphi/
 
-!3 News group:
```

Comment:
I believe we should leave this page as-is.

Reply:
Alright, I will undo the changes.